UnityProject/Assets/Scripts/Objects/StatusDisplay.cs

```diff
@@ -99,6 +102,7 @@ public void OnSpawnServer(SpawnInfo info)
 			SyncSprite(stateSync, stateSync);
 			centComm = GameManager.Instance.CentComm;
 			centComm.OnStatusDisplayUpdate.AddListener(OnTextBroadcastReceived);
+			centComm.OnAlertLevelChange += UpdateTextColor;
```

Comment:
this needs to be networked

Reply:
probably a syncvar with current colour would be best

Reply:
likie dis?